tool/gravity/cli/install.go

```diff
@@ -793,10 +793,14 @@ var InterruptSignals = signals.WithSignals(
 func NewInstallerConnectStrategy(env *localenv.LocalEnvironment, config InstallConfig, commandArgs cli.CommandArgs) (strategy installerclient.ConnectStrategy, err error) {
 	commandArgs.FlagsToAdd = append(commandArgs.FlagsToAdd,
 		cli.NewFlag("token", config.Token),
-		cli.NewBoolFlag("selinux", config.SELinux),
 		cli.NewBoolFlag("from-service", true),
 		cli.NewArg("path", config.StateDir),
 	)
+	if config.Mode != constants.InstallModeInteractive {
+		commandArgs.FlagsToAdd = append(commandArgs.FlagsToAdd,
```

Comment:
Although I initially thought to move the decision about SELinux configuration outside this method, it really has all the context to make that decision (i.e. `config.Mode`) and would require far more refactoring otherwise.